src/net/sourceforge/kolmafia/textui/command/FaxbotCommand.java

```diff
@@ -59,9 +61,13 @@ public void run( final String cmd, final String command )
 			}
 
 			Monster monster = bot.getMonsterByCommand( (String)commands.get( 0 ) );
-			FaxRequestFrame.requestFax( botName, monster, false );
-			return;
+			tried = true;
+			if (FaxRequestFrame.requestFax( botName, monster, false )) {
```

Comment:
This needs mafia-style formatting

Reply:
do you have a styling file or lint I can use?

Reply:
I found, and used, the one [posted by heeheehee](https://kolmafia.us/threads/mafia-source-code-formatting.26013/#post-161996).